@xen-orchestra/web/src/components/storage-repositories/StorageRepositoriesTable.vue

```diff
@@ -0,0 +1,193 @@
+                {{ column.value }}
+              </div>
+            </td>
+          </tr>
+        </template>
+      </VtsDataTable>
+      <VtsStateHero v-if="searchQuery && filteredSrs.length === 0" format="table" type="no-result" size="small">
+        {{ t('no-result') }}
+      </VtsStateHero>
+      <UiTopBottomTable :selected-items="0" :total-items="0">
+        <UiTablePagination v-bind="paginationBindings" />
+      </UiTopBottomTable>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import StorageRepositorySizeProgressCell from '@/components/storage-repositories/panel/table/StorageRepositorySizeProgressCell.vue'
+import type { XoSr } from '@/types/xo/sr.type'
+import type { IconName } from '@core/icons'
+import VtsDataTable from '@core/components/data-table/VtsDataTable.vue'
+import VtsIcon from '@core/components/icon/VtsIcon.vue'
+import VtsStateHero from '@core/components/state-hero/VtsStateHero.vue'
+import UiButtonIcon from '@core/components/ui/button-icon/UiButtonIcon.vue'
+import UiCheckbox from '@core/components/ui/checkbox/UiCheckbox.vue'
+import UiLink from '@core/components/ui/link/UiLink.vue'
+import UiQuerySearchBar from '@core/components/ui/query-search-bar/UiQuerySearchBar.vue'
+import UiTablePagination from '@core/components/ui/table-pagination/UiTablePagination.vue'
+import UiTitle from '@core/components/ui/title/UiTitle.vue'
+import UiTopBottomTable from '@core/components/ui/top-bottom-table/UiTopBottomTable.vue'
+import { usePagination } from '@core/composables/pagination.composable.ts'
+import { useRouteQuery } from '@core/composables/route-query.composable.ts'
+import { useTable } from '@core/composables/table.composable.ts'
+import { vTooltip } from '@core/directives/tooltip.directive.ts'
+import { noop } from '@vueuse/shared'
+import { computed, ref } from 'vue'
+import { useI18n } from 'vue-i18n'
+
+const { srs } = defineProps<{
+  srs: XoSr[]
+  hasError: boolean
+}>()
+
+const { t } = useI18n()
+
+const selectedSrId = useRouteQuery('id')
+
+const searchQuery = ref('')
+
+const filteredSrs = computed(() => {
+  const searchTerm = searchQuery.value.trim().toLocaleLowerCase()
+
+  if (!searchTerm) {
+    return srs
+  }
+
+  return srs.filter(sr => Object.values(sr).some(value => String(value).toLocaleLowerCase().includes(searchTerm)))
+})
+
+const { visibleColumns, rows } = useTable('backup-jobs', filteredSrs, {
+  rowId: record => record.id,
+  columns: define => [
+    define('checkbox', noop, { label: '', isHideable: false }),
+    define('name', record => ({ name: record.name_label, id: record.id }), { label: t('storage-repository') }),
+    define('description', record => record.name_description, { label: t('description') }),
+    define('storage-format', record => record.SR_type, { label: t('storage-format') }),
+    define('shared', record => record.shared, { label: t('shared') }),
+    define('size', record => ({ used: record.physical_usage, total: record.size }), { label: t('size') }),
```

Comment:
```suggestion
    define('space', record => ({ used: record.physical_usage, total: record.space }), { label: t('space') }),
```


In Figma the name of the Column is 'space' with the `fa-a` icon

<img width="300" height="532" alt="image" src="https://github.com/user-attachments/assets/7442fad4-dd6c-4c42-af52-f490bc8cfa4b" />